src/EditorFeatures/Core.Cocoa/NavigationCommandHandlers/FindExtensionMethodsCommandHandler.cs

```diff
@@ -62,67 +62,65 @@ protected override bool TryExecuteCommand(int caretPosition, Document document,
         private async Task FindExtensionMethodsAsync(
             Document document, int caretPosition, IStreamingFindUsagesPresenter presenter)
         {
+            var solution = document.Project.Solution;
             try
             {
                 using var token = _asyncListener.BeginAsyncOperation(nameof(FindExtensionMethodsAsync));
 
                 var (context, cancellationToken) = presenter.StartSearch(EditorFeaturesResources.Navigating, supportsReferences: true);
 
-                using (Logger.LogBlock(
-                    FunctionId.CommandHandler_FindAllReference,
-                    KeyValueLogMessage.Create(LogType.UserAction, m => m["type"] = "streaming"),
-                    cancellationToken))
+                try
```

Comment:
this is just moving to a try/finally upattern so all returns do not need to 'complete' teh searc.